src/scheduler.js

```diff
@@ -164,38 +164,27 @@ class Scheduler extends events.EventEmitter {
 
     var utf8 = s => unescape(encodeURIComponent(s));
 
-    await this.ses.sendEmail({
-      Destination: {
-        ToAddresses: [email],
-      },
-      Message: {
-        Subject: {
-          Data: utf8(`[Taskcluster Hooks] Scheduled Hook failure: ${hook.hookGroupId}/${hook.hookId}`),
-          Charset: 'UTF-8',
-        },
-        Body: {
-          Text: {
-            Data: utf8(`The hooks service was unable to create a task for hook ${hook.hookGroupId}/${hook.hookId},
-for which you are listed as owner.
-
-The error was:
-  ${err}
-
-Details:
-
-${errJson}
-
-The service will try again to create the task on the next iteration.
-
-Thanks,
-TaskCluster Automation
-
-P.S. If you believe you have received this email in error, please hit reply to let us know.`),
-            Charset: 'UTF-8',
-          },
-        },
-      },
-    }).promise();
+    this.notify.email({
```

Comment:
I don't think we need await here since we are using synchronous notify

from the [docs](https://docs.taskcluster.net/reference/libraries/taskcluster-client)
```javascript
import asyncio # Only for async 
// Create Notify client instance
import taskcluster
import taskcluster.async

notify = taskcluster.Notify(options)
# Below only for async instances, assume already in coroutine
loop = asyncio.get_event_loop()
session = taskcluster.async.createSession(loop=loop)
asyncNotify = taskcluster.async.Notify(options, session=session)

# Sync calls
notify.email(payload) # -> None`
# Async call
await asyncNotify.email(payload) # -> None
```